site/src/components/app-root/app-root.tsx

```diff
@@ -161,7 +161,15 @@ export class AppRoot {
                   </stencil-router>
 
                   <footer>
-                     <p>This work is licensed under a Creative Commons Attribution 4.0 International License.</p>
+                     <p>This work is licensed under a&nbsp;
+                        <a
+                           href="https://creativecommons.org/licenses/by/4.0/"
+                           target="_blank"
+                           rel="noopener"
+                        >
+                           Creative Commons Attribution 4.0 International License.
```

Comment:
technically we've got an MIT license

Reply:
Heh. I guess images and copy? I just copied it from the site since Carolina said we should mention Work Elections, but I can remove that license part. I'm sure there will be actual requests around disclaimers and branding once people start looking at it though. Probably a PtP logo somewhere, that sort of thing. Lmk it it's worth bothering with right now.